### ISSUE TITLE
Recalculating TADs using sep-score-prefix fails

### ISSUE THREAD

**Issue by jakevc** [deeptools/HiCExplorer#537]
When attempting to recalculate using --TAD_sep_score_prefix, hicFindTADs fails:

```
$ hicFindTADs -m ../cool/human.cool --outPrefix human_min10_max60_fdr_01 --TAD_sep_score_prefix human_min10_max60_fdr --delta 0.01 --correctForMultipleTesting fdr --thresholdComparisons 0.01 -p 16
INFO:hicexplorer.hicFindTADs:
Using existing TAD-separation score file: human_min10_max60_fdr_tad_score.bm

Traceback (most recent call last):
  File "/home/groups/hoolock2/u0/jvc/miniconda3/envs/hicexplorer/bin/hicFindTADs", line 7, in <module>
    main()
  File "/home/groups/hoolock2/u0/jvc/miniconda3/envs/hicexplorer/lib/python3.6/site-packages/hicexplorer/hicFindTADs.py", line 1372, in main
    ft.set_matrix(zscore_matrix_file, args.chromosomes)
  File "/home/groups/hoolock2/u0/jvc/miniconda3/envs/hicexplorer/lib/python3.6/site-packages/hicexplorer/hicFindTADs.py", line 406, in set_matrix
    self.hic_ma = hm.hiCMatrix(pMatrix)
  File "/home/groups/hoolock2/u0/jvc/miniconda3/envs/hicexplorer/lib/python3.6/site-packages/hicmatrix/HiCMatrix.py", line 55, in __init__
    self.correction_factors, self.distance_counts = self.matrixFileHandler.load()
  File "/home/groups/hoolock2/u0/jvc/miniconda3/envs/hicexplorer/lib/python3.6/site-packages/hicmatrix/lib/matrixFileHandler.py", line 45, in load
    return self.matrixFile.load()
  File "/home/groups/hoolock2/u0/jvc/miniconda3/envs/hicexplorer/lib/python3.6/site-packages/hicmatrix/lib/h5.py", line 35, in load
    matrix = csr_matrix(tuple([parts['data'], parts['indices'], parts['indptr']]),
KeyError: 'data'
```

Using hicexplorer in a conda environment with versions:
```
 packages in environment at /home/groups/hoolock2/u0/jvc/miniconda3/envs/hicexplorer:
hic2cool                  0.7.1                      py_0    bioconda
hicexplorer               3.4.3                      py_0    bioconda
hicmatrix                 12                         py_0    bioconda
```

**Comment by agalitsyna** [comment]
Hi, I think I've encountered a problem that might be related. 
I run hicFindTADs on .cool file and do multiple runs (so that prefix remains the same). However, for the second run, when HiCExplorer recognizes the prefix and uses existing files, I got the following error: 

```
hicFindTADs --matrix dataset.10000.cool --outPrefix tmp --correctForMultipleTesting fdr --minDepth 30000 --maxD
epth 1000000 --step 10000 --delta 0.01 -p 20
INFO:numexpr.utils:Note: detected 80 virtual cores but NumExpr set to maximum of 64, check "NUMEXPR_MAX_THREADS" environment variable.
INFO:numexpr.utils:Note: NumExpr detected 80 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
INFO:numexpr.utils:NumExpr defaulting to 8 threads.
INFO:hicexplorer.hicFindTADs:
Found existing TAD-separation score file: tmp_tad_score.bm

INFO:hicexplorer.hicFindTADs:This file will be used

INFO:hicmatrix.lib.h5:No h5 file. Please check parameters concerning the file type!```

**Comment by jakevc** [comment]
see #377

**Comment by joachimwolff** [comment]
Implemented with version 3.5.
